ui/src/notebooks/components/panel/InsertCellMenu.tsx

```diff
@@ -0,0 +1,87 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+
+// Components
+import {
+  Button,
+  ComponentColor,
+  FlexBox,
+  FlexDirection,
+  AlignItems,
+  ComponentSize,
+} from '@influxdata/clockface'
+
+// Constants
+import {NotebookContext} from 'src/notebooks/context/notebook'
+import {PIPE_DEFINITIONS} from 'src/notebooks'
+
+import {event} from 'src/notebooks/shared/event'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+interface Props {
+  index: number
```

Comment:
could we just make this an optional prop, and then it's the same thing as the addbuttons?